index.js

```diff
@@ -32,33 +23,29 @@ client.on("message", (message) => {
     message.reply(`Your access code is ${message.guild.id}`);
   } else if (message.content === `${prefix}reset`) {
     FirebaseHelpers.resetDB();
-  } else if (
-    message.content.substr(0, prefix.length + 8) === `${prefix}entries `
-  ) {
+  } else if (message.content.startsWith(`${prefix}entries`)) {
     const entriesFunction = async () => {
       let response = await FirebaseHelpers.numEntries(
         message.guild.name,
-        message.content.substr(prefix.length + 8)
+        message.content.substr(prefix.length + 7)
       );
-      message.reply(response);
+      message.reply(JSON.stringify(response));
     };
     entriesFunction();
   } else if (!message.author.bot) {
-    const existsFunction = async () => {
+    const checkIfServerExistsAndIncrementOrCreate = async () => {
       let exists = await FirebaseHelpers.serverExists(message.guild.id);
       if (exists) {
-        message.reply("You exist");
-        FirebaseHelpers.addMessage(message.guild.name);
+        FirebaseHelpers.addMessage(message.guild.name, message.member.user.tag);
       } else {
-        message.reply("You don't exist");
         FirebaseHelpers.addServer(
           message.guild.id,
           message.guild.name,
           message.member.user.tag
         );
       }
     };
-    existsFunction();
+    checkIfServerExistsAndIncrementOrCreate();
```

Comment:
same as above comment, just make the whole func `async` so you don't have to do this wrapper function thing